### PR TITLE
Add github action to automatically bump version on PR

### PR DESCRIPTION
**Pull request by Waelthus**
add a github action to increase the patch version on every push to master, currently just a skeleton, needs completion and testing.

### CODE REVIEW

**Comment by Waelthus** [comment]
adresses #1010

File: .github/workflows/bump_version.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: Pytest
```
Comment (iprafols) [inline .github/workflows/bump_version.yml:1]: why this name? Maybe something more descriptive?

File: .github/workflows/bump_version.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+    - name: Install bump2version
+      run:  pip install bump2version
+    - name: increase version
+      run: bump2version patch py/picca/_version.py --no-tag
```
Comment (iprafols) [inline .github/workflows/bump_version.yml:16]: How do we distinguish between major/minor/patches?

**Comment by Waelthus** [comment]
this is just a draft, didn't get to complete it yet. So yes, not fully done yet ;-)

**Comment by iprafols** [comment]
ah, sorry, didn't see the draft bit

**Comment by Waelthus** [comment]
note that this will auto-generate tags on minor/major version bumps which then should get auto-submitted to pypi via the other action. So the only thing to be done when we actually want to create new versions is merging the last PR using [bump minor] or [bump major] and creating the release on github for documentation purposes

**Review by iprafols — APPROVED** [review]
Looks fine

**Pull request merged**